Sources_v3/StreamChatUI/ChatChannel/ChatChannelMessageComposerView.swift

```diff
@@ -4,13 +4,16 @@
 
 import UIKit
 
-open class ChatChannelMessageComposerView<ExtraData: UIExtraDataTypes>: UIInputView {
+open class ChatChannelMessageComposerView<ExtraData: UIExtraDataTypes>: UIInputView,
+    UIConfigProvider,
+    Customizable,
+    AppearanceSetting {
     // MARK: - Properties
     
-    public var buttonHeight: CGFloat = 20
+    open var attachmentsViewHeight: CGFloat = .zero
+    open var titleViewHeight: CGFloat = .zero
+    open var stateIconHeight: CGFloat = .zero
```

Comment:
I don't understand what are these things for :)

Reply:
We talked about this in Slack, but basically the only change is that they are `public` rn 🤷‍♂️
Am I still missing something?